### PR TITLE
[Lexical][Meta] Add meta intern static docs config

### PR DESCRIPTION
**Pull request by Sahejkm**
## WHAT
- Add meta intern static docs config
- Fix broken JSX rendering in MD files caused due to preprocessor added for markdown validations

## WHY
- needed to sync and build intern static docs in meta for lexical

### CODE REVIEW

File: packages/lexical-website/docusaurus.config.js [hunk 1]
```diff
@@ -225,15 +225,6 @@ const config = {
...
 
-  markdown: {
-    format: 'md',
-    preprocessor: ({fileContent}) =>
-      fileContent.replaceAll(
-        'https://stackblitz.com/github/facebook/lexical/tree/main/',
-        STACKBLITZ_PREFIX,
-      ),
-  },
-
```
[the remark this answers is not in the document]
Comment (etrepum) [reply packages/lexical-website/docusaurus.config.js:236]: have you tried just removing the `format: 'md',`?

Comment (etrepum) [inline packages/lexical-website/docusaurus.config.js:236]: it looks like this update to docusaurus-plugin-internaldocs-fb is switching from a markdown to mdx, so removing that configuration override should resolve the problem without throwing away the preprocessor which is useful for testing new examples in preview environments.

```js
  markdown: {
    preprocessor: ({fileContent}) =>
      fileContent.replaceAll(
        'https://stackblitz.com/github/facebook/lexical/tree/main/',
        STACKBLITZ_PREFIX,
      ),
  },
```

Comment (Sahejkm) [reply packages/lexical-website/docusaurus.config.js]: this does the job indeed. Thanks @etrepum Bob!

File: packages/lexical-utils/src/index.ts [hunk 2]
```diff
@@ -131,9 +131,9 @@ export function isMimeType(
  *  3. Order aware (respects the order when multiple Files are passed)
  *
  * const filesResult = await mediaFileReader(files, ['image/']);
- * filesResult.forEach(file => editor.dispatchCommand('INSERT_IMAGE', {
+ * filesResult.forEach(file => editor.dispatchCommand('INSERT_IMAGE', \\{
  *   src: file.result,
- * }));
+ * \\}));
```
Comment (potatowagon) [inline packages/lexical-utils/src/index.ts:136]: what is \\ this for?

**Review by potatowagon — APPROVED** [review]

File: packages/lexical-utils/src/index.ts (hunk 2 above)
Comment (Sahejkm) [reply packages/lexical-utils/src/index.ts:136]: Fixing parsing error for mdx , it misunderstands it as code

Comment (etrepum) [reply packages/lexical-utils/src/index.ts:136]: MDX is implicitly JSX anywhere that could possibly make sense (it's a weird hybrid of markdown and jsx), so wherever you have `{` or `<` outside of some other markdown context like backticks or a code block it starts parsing as a JSX child expression or a JSX tag.

Comment (potatowagon) [reply packages/lexical-utils/src/index.ts:136]: i see

**Pull request merged**